docs/Reference-for-template.json.md

```diff
@@ -75,8 +75,8 @@ A symbol for which the config provides literal and/or default values.
 |`dataType`|	Supported values: <br />- `bool`: boolean type, possible values: `true`/`false`. <br />- `choice`: enumeration, possible values are defined in `choices` property.<br />- `float`: double-precision floating format number. Accepts any value that can be parsed by `double.TryParse()`.<br />- `int`/`integer`: 64-bit signed integer. Accepts any value that can be parsed by `long.TryParse()`.<br />- `hex`: hex number. Accepts any value that can be parsed by `long.TryParse(value.Substring(2), NumberStyles.HexNumber, CultureInfo.InvariantCulture, out long convertedHex)`.<br />- `text`/`string`: string type.<br />- `<any other>`: treated as string.
 |`defaultValue`|The value assigned to the symbol if no parameter is provided by the user or host.|
 |`binding`|The name of the host property to take the value from.|	
-|`replaces`|The text to replace with the value of this symbol.|	 
-|`fileRename`|The element defines the portion of file names which will be replaced by symbol value.| 
+|`replaces`|The text to be replaced by symbol value in template files content|	 
+|`fileRename`|The portion of template filenames to be replaced by symbol value.| 
```

Comment:
...`by the symbol value.` OR ...`by the symbol's value.`

Reply:
as far as I know possessive case should not be used with objects. Tbh I can barely remember I saw it used in documentation.